manager/api/views.py

```diff
@@ -106,6 +116,92 @@ def logout(request):
     )
 
 
+class IPABackend1(LDAPBackend):
+    settings_prefix = "AUTH_LDAP_1_"
+    successful_login = False
+
+    def authenticate_ldap_user(self, ldap_user, password):
+        user = ldap_user.authenticate(password)
+        if user:
+            IPABackend1.successful_login = True
+        return user
+
+
+class IPABackend2(LDAPBackend):
+    settings_prefix = "AUTH_LDAP_2_"
+    successful_login = False
+
+    def authenticate_ldap_user(self, ldap_user, password):
+        user = ldap_user.authenticate(password)
+        if user:
+            IPABackend2.successful_login = True
+        return user
+
+
+class IPABackend3(LDAPBackend):
+    settings_prefix = "AUTH_LDAP_3_"
+    successful_login = False
+
+    def authenticate_ldap_user(self, ldap_user, password):
+        user = ldap_user.authenticate(password)
+        if user:
+            IPABackend3.successful_login = True
+        return user
+
+
+class LDAPLogin(APIView):
+    """
+    Class to authenticate a user via LDAP and
+    then creating a login session
+    """
+
+    authentication_classes = ()
+
+    permission_classes = [AllowAny]
+
+    def post(self, request):
+        """
+        Api to login a user
```

Comment:
Please add a description of what the method does:
1. It authenticates to an LDAP server, if none is found the default login is used
2. It searches for the love_ops group and if the authenticated user is present, command permissions are added.